### ISSUE TITLE
Display a list of inhibitors before powering off or logging out

### ISSUE THREAD

**Issue by iBelieve** [papyros/papyros-shell#124]
##

**Comment by ghost** [comment]
Again, ps -o name? -ppid _pid of session_. and filter all of the system processes....

**Comment by iBelieve** [comment]
@Schwertspize No, the list of inhibitors comes from logind, but we'll also add our own inhibitors for non-power related locks (similar to what GNOME does).
